how-to/debug-buildfarm-builder.rst

```diff
@@ -1,30 +1,47 @@
-Debug Build Farm
-================
+Debug Build Farm on qastaging
+=============================
 
-Access a builder in qastaging
------------------------------
+Access a builder
+----------------
 
 The builders are ephemeral, meaning they fully reset after a build.
 Therefore, to access the builder, one should either:
 
 * access it during the duration of a build.
 
-* set the builder to ``manual`` after starting a build so that it doesn't
+* ``disable`` the builder after starting a build so that it doesn't
   automatically trigger the reset when the build finishes.
 
+.. note::
 
-Set a builder to manual
-~~~~~~~~~~~~~~~~~~~~~~~
+   You should wait until you see some output in the build log before you
+   ``disable`` the builder, otherwise the build might not get dispatched
+   properly.
 
-This requires permissions granted by being part of the ``~admins`` team.
-To anoint yourself to be part of that team in qastaging, see
-:doc:`./manage-users`.
 
-1. Go to https://qastaging.launchpad.net/builders.
+Disable a builder
+~~~~~~~~~~~~~~~~~
 
-2. Click on the builder you want to set to manual.
+You need to have the ubuntu-archive-tools installed.
```

Comment:
Having a link to ubuntu-archive-tools helps here. For the very first time, I was confused by what is it - https://code.launchpad.net/ubuntu-archive-tools